tests/test_azure_batch_executor.py

```diff
@@ -0,0 +1,20 @@
+import sys
+import os
+
+sys.path.insert(0, os.path.dirname(__file__))
+
+from common import *
+
+
+def test_az_batch_executor():
+    bau = os.getenv("AZ_BATCH_ACCOUNT_URL")
+    prefix = os.getenv("AZ_BLOB_PREFIX")
+    wdir = dpath("test_azure_batch")
+    run(
+        path=wdir,
+        default_remote_prefix=prefix,
+        container_image="jakevc/snakemake",
```

Comment:
Why is a special container image needed?

Reply:
It shouldn't I think we need to back into base snakemake image